README.md

```diff
@@ -33,12 +33,21 @@ TBD-welcome-paragraph
 
 ## How to start this course
 
+<!-- For TBD-generate, run in JavaScript:
+'https://github.com/new?' + new URLSearchParams({
+  template_owner: 'TBD-organization',
+  template_name: 'TBD-course-name',
+  owner: '@me',
+  name: 'TBD-organization-TBD-course-name',
+  description: 'My clone repository',
+  visibility: 'public',
+}).toString()
+-->
+
+[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](TBD-generate)
+
 1. Right-click **Start course** and open the link in a new tab.
-   <br />[![start-course](https://user-images.githubusercontent.com/1221423/218596841-0645fe1a-4aaf-4f51-9ab3-8aa2d3fdd487.svg)](TBD-generate)
-2. In the new tab, follow the prompts to create a new repository.
-   - For owner, choose your personal account or an organization to host the repository.
-   - We recommend creating a public repository—private repositories will [use Actions minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions).
-   ![Create a new repository](https://user-images.githubusercontent.com/1221423/218594143-e60462b6-9f2a-4fa3-80de-063ac5429aab.png)
+2. In the new tab, follow the prompts to create a new repository. For owner, we recommend your personal account.
```

Comment:
> For owner, we recommend your personal account.

For now `@me` isn't functional, but it hopefully will be in the coming weeks.

Reply:
Do we mention Actions minutes anywhere else? If not, we might want to move that note elsewhere so folks aren't surprised. 

Also, @heiskr can you share context on not recommending an org anymore?

```suggestion
2. In the new tab, follow the prompts to create a new public repository. For owner, choose your personal account or an organization to host the repository.
```

Reply:
I removed the public bit since it autopopulates with the query string input. We can keep the org part, though once `@me` starts working I was thinking we'd remove that sentence as well to reduce the load.

Reply:
I don't think we mention Actions minutes anywhere, either, so definitely agree with adding that back.